v1.1: Use wildcard (`*`) when creating tenant tokens (#2210)
meilisearch/documentation: learn/security/tenant_tokens.md

}
```

> You can also use the `*` wildcard by adding it at the end of an index name string. This allows the tenant token to access all index names starting with that string.

Reviewer: ```suggestion
You can also use the `*` wildcard by adding it at the end of a string. This allows the tenant token to access all index names starting with that string.
```

It is not an index name at this point but just the prefix of one 😊

The following example queries across all indexes starting with the string `medical_records` and returns documents whose `user_id` equals `1`:

```json
{

>   "medical_records*": {

Reviewer: Can we do a more obvious example, please? With `medical_*` for example?
